src/installer.ts

```diff
@@ -123,8 +127,70 @@ function normalizeVersion(version: string): string {
     //append minor and patch version if not available
     return version.concat('.0.0');
   } else if (versionPart[2] == null) {
+    // handle beta and rc: 1.10beta1 => 1.10.0-beta1, 1.10rc1 => 1.10.0-rc1
+    if (versionPart[1].includes('beta') || versionPart[1].includes('rc')) {
+      versionPart[1] = versionPart[1]
+        .replace('beta', '.0-beta')
+        .replace('rc', '.0-rc');
+      return versionPart.join('.');
+    }
+
     //append patch version if not available
     return version.concat('.0');
   }
   return version;
 }
+
+async function determineVersion(version: string): Promise<string> {
+  if (!version.endsWith('.x')) {
```

Comment:
Could we do something similar to `normalizeVersion` here, but instead of appending 0s at the end, append a `.x`? What I'm after is, I want people to just be able to specify `1.10` and have that resolve to `1.10.x`, since referring to versions like that is pretty common.

Reply:
Thanks for the suggestion, I've pushed an update to handle this.